src/grip/index.tsx

```diff
@@ -0,0 +1,9 @@
+import styled from 'styled-components'
```

Comment:
Hey, we are changing the way we export our files on ui-library. Since this is a new component, could you please refeer to `ItemBigData` to see the template?
TLDR;
- index will export only the component (not a styled-component)
- styled-component is declared "inside" the component
- you can use an external file .style if you will
- stories also changed, you can write it on CSF or MDX

Reply:
Thanks for the tips, I'll have a look :)

Reply:
Updated the PR with the new docs format and removed the unused style from index :)